src/utils/index.js

```diff
@@ -19,6 +20,128 @@ function logInfo(message) {
   console.log(blue(message));
 }
 
+function logHelp(command) {
+  const mainHelp = `
+Usage:
+  blazepack   Start dev server on existing project
+  blazepack   [options]  [command]
+
+Options:
+  --version   Output the current version
+  --port <p>  Specify port <p> for dev server
+
+Commands:
+  create          Create project from Template
+  start           Create project from Template & Start the dev server
+  clone           clone an existing codesandox
```

Comment:
Typo in codesandbox & capitalizing first letter as done in every description
```suggestion
  clone           Clone an existing codesandbox
```